### PR TITLE
LP-816 | Update UI Flutter

### PR DESCRIPTION
**Pull request by sevasvalveryfi**
**Description**
- Please, let's conduct a thorough review of our app and update everything related to the UI, ensuring consistency in colors, fonts, shadows, and background colors.

**Solution**
- This Pr add new UI Design for Flutter project
- Add multiple documents Scanning options
- New loading animation
- New Extracted data format UI
- New JsonViewer format UI
- New Logic for handle Veryfi Lens data

**Evidences**

extracted
json
loading
mainmenu
settings

**Notes**
- This Pr closes LP-816 Jira ticket project

### CODE REVIEW

**Review by alejouribesanchez — CHANGES_REQUESTED** [review]
launcher icon should be keep it

**Comment by beresmo** [comment]
some feedback:

> Tabs

- no following the green color
- No consistency between lowercase and uppercase

> Card views

- No following background color

**Comment by beresmo** [comment]
- Missing receipt preview screenshot
- Missing settings screenshot
- How do I navigate after scanning a receipt?
- The extracted data should be left and right, left the tag and right the information.

**Comment by beresmo** [comment]
What does dark mode look like?

**Comment by decim92** [comment]
We don't need anything related to other non mobile platforms like web, linux or macOS files or folders

**Comment by decim92** [comment]
Please upload at least one screenshot of running in iOS simulator

File: .gitignore [hunk 1]
```diff
@@ -44,9 +41,3 @@ app.*.map.json
 /android/app/debug
 /android/app/profile
 /android/app/release
-
-# Env variables
-.env
```
Comment (decim92) [inline .gitignore:49]: I see we're still using dotenv so why this line about ignoring `.env` file should not be removed

**Comment by decim92** [comment]
I'd rather have all new widgets in individual files, instead of a massive `main.dart` file

File: .gitignore [hunk 2]
```diff
@@ -44,9 +41,3 @@ app.*.map.json
...
-
-#pubspeck.lock file
-pubspec.lock
```
Comment (decim92) [inline .gitignore:52]: This one also should be ignored, unless there's something that forces us to upload it, the `pubspec.yaml` will define latest dependencies versions (or if you want specific versions, they should be defined in the `pubspec.yaml`
Also don't forget to remove `pubspec.lock` from git history

Comment (sevasvalveryfi) [reply .gitignore]: Done

File: .gitignore (hunk 1 above)
Comment (sevasvalveryfi) [reply .gitignore]: Done

**Comment by sevasvalveryfi** [comment]
> I'd rather have all new widgets in individual files, instead of a massive `main.dart` file

Only Android is part of this scope, for another side, i need setting up all IOS env for flutter like xcode cocoapods ans so on, for now is Android

**Comment by sevasvalveryfi** [comment]
> We don't need anything related to other non mobile platforms like web, linux or macOS files or folders

Done

**Comment by sevasvalveryfi** [comment]
> launcher icon should be keep it

Done

**Review by beresmo — APPROVED** [review]

**Pull request closed without merging**